config/singleton_enclave_config.toml

```diff
@@ -78,6 +78,9 @@ enclave_type = "SWE"
 [WorkerConfig]
 # Id of worker in plain text
 worker_id = "singleton-worker-1"
+# Supported workloads for this worker. This can be overridden by passing
+# values in command line for enclave manager using --workloads option.
+workloads = "echo-result,heart-disease-eval,inside-out-eval,simple-wallet"
```

Comment:
It is static way of representing association of workloads with worker. It is good start with it. But in future we should make dynamic way to registering workloads to worker. Meaning as part of workload registration we should notify the worker's untrusted python, so that it can maintain workloads items in memory/DB.